modules/ROOT/pages/userlookup.adoc

```diff
@@ -0,0 +1,191 @@
+= Using the User Lookup API
+:description: TinyMCE User Lookup provides an API to identify the current user and retrieve user details via integration-defined methods.
+:keywords: userLookup, fetchUsers, userId, avatar, user metadata
+
+The {productname} User Lookup API enables integrations to retrieve and cache user details (such as names and avatars) and identify the current user within the editor.
+
+This is useful when building features that rely on user context, such as commenting, or displaying a list of elements container user information, and lets you manage user data efficiently with built-in caching.
+
+== Using the User Lookup API
+
+Follow the steps below to configure and use the `userLookup` API.
```

Comment:
This isn't using the API, it's implementing it for other code to use.
```suggestion
== Implementing the User Lookup API

Follow the steps below to configure the `UserLookup` API.
```

Reply:
Removed in recent commit.